aws/orangehrm

```diff
@@ -54,10 +54,7 @@ export ORANGEHRM_WEBROOT_VOLUME="orangehrm_orangehrm-webroot"
 export ORANGEHRM_DB_VOLUME="orangehrm_orangehrm-db"
 
 # Update URL
-export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/main/aws/compose.yml"
-
-# License changed version
-export NEW_LICENSE_VERSION="5.6"
+export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/orangehrm-os-cloud-packages/main/aws/compose.yml"
```

Comment:
hope all places changed related to repo name change

Reply:
changed